smart_open/smart_open_lib.py

```diff
@@ -341,8 +341,10 @@ def open(
     #
     try:
         binary_mode = {'r': 'rb', 'r+': 'rb+',
+                       'rt': 'rb', 'rt+': 'rb+',
```

Comment:
What about `wt` and `wt+`?

Are there any other modes we're missing?

Reply:
Required modes: `rb`, `ab`, `wb`?